oat-sa/extension-tao-testqti-previewer
# Fix/csrf token in test preview

In the Results, when trying to review an item it appears the token was sent but no new token was issued.

How to test : 
1. Go to _Results_ -> _View_
![image](https://user-images.githubusercontent.com/468620/60300429-3bf5fc00-992f-11e9-9f2c-b09dcf12c979.png)
2. Open the item result review a couple of time and the following error message appears : 
![image](https://user-images.githubusercontent.com/468620/60300492-6b0c6d80-992f-11e9-94cb-9ea29481ef9b.png)
